Kernel/Devices/Device.h

```diff
@@ -57,6 +57,9 @@ class Device : public File {
     uid_t uid() const { return m_uid; }
     uid_t gid() const { return m_gid; }
 
+    // ^Device
+    virtual mode_t permission_flags() const = 0;
```

Comment:
No point in `// ^Device` inside `Device` itself 😀

Also perhaps call it just `mode()`?

Reply:
I guess it could be `mode`, or maybe `required_mode`?